source/release-notes/1.8.txt

```diff
@@ -21,9 +21,91 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.8, ``mongosync`` includes an optional embedded
+verifier that performs a series of checks against the source and 
+destination clusters to verify a successful sync.
+
+For more information, see :ref:`c2c-embedded-verifier`.
+
+Other Notes
+~~~~~~~~~~~
+
+Updates to ``mongosync`` ``/progress`` API endpoint:
+
+- The ``/progress`` endpoint now returns a top-level boolean 
+  ``success`` field.
+
+- The ``/progress`` endpoint now returns the integer 
+  ``totalEventsApplied`` field, which approximates the number of 
+  change events applied to a destination cluster by an instance of 
+  ``mongosync``.
+
+- The ``/progress`` endpoint now reports per-partition progress 
+  tracking with the following changes:
+
+  - Calling ``/progress`` before ``mongosync`` reaches the collection 
+    copy phase will return 0 for ``estimatedCopiedBytes`` and 1 for 
+    ``estimatedTotalBytes``.
+
+  - ``estimatedTotalBytes`` may change throughout the collection copy 
+    phase if documents are inserted or deleted on the source cluster.
+
+  - ``estimatedCopiedBytes`` will never be greater than 
+    ``estimatedTotalBytes``. Progress will reach 100% at the end of 
```

Comment:
(tense)
Progress reaches 100%...

I'll stop marking. See if you can eliminate will, globally.

Reply:
Done